service/provider/src/main/java/gov/va/vro/service/provider/camel/MasIntegrationRoutes.java

```diff
@@ -126,6 +126,7 @@ private void configureMasProcessing() {
         .onPrepare(auditProcessor(routeId, "Started claim processing."))
         .process(convertToMasProcessingObject())
         .setProperty("diagnosticCode", simple("${body.diagnosticCode}"))
+        .setProperty("idType", simple("${body.idType}"))
```

Comment:
I did not want to risk breaking other processing nor did I want defaults set in multiple places if I could at all avoid it. Thus we pass along the property off the MasProcessing object here.